### ISSUE TITLE
 "no-console": 0 , it's not working v5.1.0

### ISSUE THREAD

**Issue by zigang93** [eslint/eslint#10616]
**Tell us about your environment**

* **ESLint Version:** v5.1.0
* **Node Version:** v10.4.1
* **npm Version:** 6.1.0

**What parser (default, Babel-ESLint, etc.) are you using?**  eslint:recommended

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
    "env": {
        "browser": true
    },
    "extends": "eslint:recommended",
    "parserOptions": {
        "ecmaVersion": 5
    },
    "rules": {
        "indent": [
            "error",
            4
        ],
        "linebreak-style": [
            "error",
            "unix"
        ],
        "quotes": [
            "error",
            "single"
        ],
        "semi": [
            "error",
            "always"
        ],
        "no-console": 0
    }
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```
eslint index.js

```

```bash

```

**What did you expect to happen?**
`Unexpected console statement  no-console` should be remove. but still show up.

**What actually happened? Please include the actual, raw output from ESLint.**
`Unexpected console statement  no-console`

**Comment by zigang93** [comment]
I just solve the problem, closed issue.
